### PR TITLE
 analyze table sorting added

### PR DESCRIPTION
**Pull request by bergano65**

### CODE REVIEW

File: CogsMinimizer/Views/Subscription/Analyze.cshtml [hunk 1]
```diff
@@ -16,33 +21,35 @@
 <br/>
 <br/>
 
-<table class="table table-hover table-striped">
+<table class="table table-hover table-bordered table-striped table-condensed sortable" id="resourcesTable">
+    <thead>
     <tr>
-        <th>
+            <th id="nameColumn" data-defaultsign="AZ">
```
Comment (shklar) [inline CogsMinimizer/Views/Subscription/Analyze.cshtml]: Does this mean that we can only sort by the resource name?
I would also like to be able to sort by the resource group name, or owner, or date (expiration or found).
Ideally, I would like to be able to sort by any of the columns.

If this is more complicated then please submit as is, and add sorting capabilities for the other columns in a separate pull request.

Comment (shklar) [inline CogsMinimizer/Views/Subscription/Analyze.cshtml]: I have merged this change.
Please add sorting by other columns separately.

Comment (shklar) [inline CogsMinimizer/Views/Subscription/Analyze.cshtml]: Looks like it already is sortable by any column.
Great work Evgeny!

**Comment by bergano65** [comment]
It’s interesting world of asp.net, jquery etc. thanks for possibility to learn it getting payment simultaneously ☺

**Pull request merged**